src/snowflake/cli/plugins/nativeapp/commands.py

```diff
@@ -69,6 +81,39 @@ def app_init(
     )
 
 
+@app.command("list-templates", hidden=True)
```

Comment:
Today we have feature flags. Have you consider using them?

Reply:
I believe this was clarified offline, please see the slack thread of the PR for the full discussion, but to summarize:
- For now, we intend to make it available for our IDE plugin, but not expose directly it to users
- We don't want users to have to enable a feature flag before the IDE plugin can work
- It doesn't fit the use case of an experimental flag since this command doesn't change the behaviour of any existing commands nor is it interacting with Snowflake

Please let me know if this works for you :)